src/SortableContainer/index.js

```diff
@@ -58,9 +59,9 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 			};
 		}
 		componentDidMount() {
-			let {contentWindow} = this.props;
+			let {contentWindow, getContainer} = this.props;
 
-			this.container = ReactDOM.findDOMNode(this);
+			this.container = (typeof getContainer == 'function') ? getContainer(this.getWrappedInstance()) : ReactDOM.findDOMNode(this);
```

Comment:
I was on the fence about whether this should return a React element or a DOM node. Thoughts?